docs/runbooks/VirtHandlerRESTErrorsBurst.md

```diff
@@ -1,17 +1,55 @@
+<!-- Edited by Jiří Herrmann, 7 Nov 2022 -->
+
 # VirtHandlerRESTErrorsBurst
 
 ## Meaning
 
-More than 80% of the rest calls failed in virt-handler for the last 5 minutes
+More than 80% of REST calls failed in `virt-handler` in the last 5 minutes. This alert usually indicates that the `virt-handler` pods cannot connect to the API server.
+
+This error is most frequently caused by one of the following problems:
+
+- The API server is overloaded, which causes timeouts. To verify if this is the case, check the metrics of the API server, and view its response times and overall calls.
+
+- The `virt-handler` pod cannot reach the API server. This is commonly caused by DNS issues on the node and networking connectivity issues.
 
-virt-handler has potentially fully lost connectivity to the apiserver. Running workloads on the affected node will not be impacted but status updates will not be propagated and actions like migrations can not take place.
+## Impact
+
+Running workloads on the affected node are not impacted, but status updates are not propagated and node-related actions, such as migrations, fail.
```

Comment:
This sentence is a bit too long to be readable. I recommend that you break it into two sentences, with the same structure as your other REST API PR:

blah blah blah might fail. 
However, blah blah blah is not affected.

That way, the user sees the most important item first (what won't work).